USAGE.md

```diff
@@ -281,6 +283,26 @@ The `--ref` argument allows for passing a specific `git` tag (e.g. `v.1.5.0`) or
 $ python scripts/generator.py --ref v1.5.0
 ```
 
+The `--ref` argument only loads field definitions from the [`./schemas`](./schemas) subdirectory in the target `git` reference. This allows for merging custom fieldsets (outside the git ref tree) using `--include`.
+
+Here's an example merging ECS `v1.6.0` fields with custom fields maintained in a separate `myproject` directory:
+
+```
+$ python scripts/generator.py --ref v1.6.0 --include ../myproject/fields/custom`  --out ../myproject/out
+```
+
+There one is one exception supporting the [experimental](experimental/README.md) fields. If using `--ref` and including the `experimental/schemas` directory, the experimental fields will also load from the specified git ref. This example would merge ECS v1.7.0 fields with the v1.7.0 experimental fields:
+
+```
+$ python scripts/generator.py --ref v1.7.0 --include experimental/schemas --out ../myproject/out
+```
+
+Custom fields can also be passed using `--include` with the experimental ones. This command merges v1.7.0 fields, v1.7.0 experimental fields, and custom fields from `../myproject/fields/custom`:
```

Comment:
I like this explicit call out, but given the rewritten intro paragraph I'm suggesting above, I would move it below the example, as bullet points.

This way we can spell out what's happening in a bit more details. Perhaps something like this:

```suggestion
The command above will produce artifacts based on:

* main ECS field definitions as of branch 1.7
* experimental ECS changes as of branch 1.7
* custom fields in `../myproject/fields/custom` as they are on the filesystem
```